RecoPixelVertexing/PixelLowPtUtilities/src/ClusterShapeTrackFilter.cc

```diff
@@ -1,25 +1,19 @@
-#include "RecoPixelVertexing/PixelLowPtUtilities/interface/ClusterShapeTrackFilter.h"
```

Comment:
Did you intend to reformat also the irrelevant (for us) files?

Reply:
Ok. Would be worth of the effort to propagate those back to CMSSW master?

Reply:
Yes, I usually try to make a PR for the CMSSW master whenever I rebase our branch on a new release (and 10.2.2 just came out...)